### DISCUSSION TITLE
Correct Ableton Live Version

### DISCUSSION THREAD

**Discussion by m0e33** [Ghadjeres/piano-inpainting-application#8] [Q&A]
Hey all, 

I tried to import the amxd File but I always get the following "Error":
`This device is not available in this version of Live` (see attached image)

<img width="486" alt="image" src="https://user-images.githubusercontent.com/32243674/172025975-37813e84-4d92-4bf6-866b-093e1016d299.png">

My Ableton Version is 11 Standard: Build 2021-11-04.

What version did you use when building PIA? Is there anything I can do to get it running?

**Comment by Ghadjeres** [comment]
When I tested with the trial version of Live 11, I got something like #7 
Just a prompt that you need to accept
I'll investigate.
